### PR TITLE
slice: update  master with 8.2 changes

### PR DESCRIPTION
**Pull request by chewitt**
This updates master to use the slice.service add-on and other minor changes present in the current LE 8.2 release. For some reason symlinking `linux.arm.conf` errors the build script I use so I moved the link up to the `linux` folder which works fine.

### CODE REVIEW

File: packages/linux-firmware/slice-firmware/package.mk [hunk 1]
```diff
@@ -33,13 +33,13 @@ PKG_AUTORECONF="no"
 
 make_target() {
   if [ "$DEVICE" = "Slice3" ]; then
```
Comment (MilhouseVH) [inline packages/linux-firmware/slice-firmware/package.mk]: No need to check $DEVICE here, as we only build slice-firmware for suitable devices.

Comment (chewitt) [reply packages/linux-firmware/slice-firmware/package.mk]: Slice and Slice3 have different firmware .. hence the check

Comment (MilhouseVH) [reply packages/linux-firmware/slice-firmware/package.mk]: Ah yes...

**Comment by chewitt** [comment]
I've changed `/dev/mmcblk0p1` to `mmcblk1p1` for Slice as a temp. measure, because at some point since the 4.9 kernel things switched and this causes boot failure on updating from any 8.x releases. I'll need to figure out how to keep the original naming.. but that will take me a while to figure out so there's no point holding this.

**Comment by chewitt** [comment]
@MilhouseVH .. I dropped the `scripts/mkimage` change. I'll need to ferret in the dts files

**Pull request merged**